assets/src/edit-story/components/panels/design/sizePosition/sizePosition.js

```diff
@@ -43,6 +44,10 @@ import usePresubmitHandlers from './usePresubmitHandlers';
 import { getMultiSelectionMinMaxXY, isNum } from './utils';
 import { MIN_MAX, DEFAULT_FLIP } from './constants';
 
+function getStickerAspectRatio(element) {
+  return stickers?.[element?.sticker?.type].aspectRatio || 1;
```

Comment:
If the aspect ratio is `0` then should we return `0`? I feel like having a `0` aspect ratio is not really likely anyways lol

Reply:
haah. didn't consider that case, but yea I don't think it would be visible to the user if that were the case